messaging/usecase/room_usecase.go

```diff
@@ -67,41 +111,32 @@ if err!=nil {
 		// should we fill all students  ?? data
 		Room.Students=room.Students ;
 		Room.Deleted=room.Deleted ;
-
 	}
 
 	return studentChatRooms,nil
 }
 
-func (u *roomUseCase) EditChatRoomParticipants( room *domain.ChatRoom) error{
-
-	_,err:=u.rr.GetRoom(room.RoomID)
-	if err==nil {
-		return errors.NewConflictError(fmt.Sprintf("Room with ID %s already exists", room.RoomID))
+// DeleteRoom should delete room for all users in chat.student_rooms and delete room from chat.room
+func (u *roomUseCase) DeleteRoom( userID string, roomID string) error {
+	room,err := u.rr.GetRoom(roomID)
```

Comment:
are we sure this is the only reason for the error? It could be some other error. Also, does it return an error if there are no results found or returns an empty room struct and nil error?

Reply:
from what I could test, which is searching for a nonexistant roomid, I get a 500 not found error. You're right there could be other errors that I'm unaware of, so I'll just return the error with it's original message and code